recipes-debian/python/python-psutil.inc

```diff
@@ -0,0 +1,25 @@
+# base recipe: meta-openembedded/meta-python/recipes-devtools/python/python3-psutil.inc
```

Comment:
This should be `python-psutil.inc`, not `python3-psutil.inc`

Reply:
Ouch!

Reply:
Updated.